src/SFML/Window/OSX/WindowImplCocoa.mm

```diff
@@ -436,6 +436,22 @@ void showMouseCursor()
 }
 
 
+////////////////////////////////////////////////////////////
+void WindowImplCocoa::setMinimumSize(const Vector2u& minimumSize)
+{
+    AutoreleasePool pool;
+    [m_delegate setMinimumSize:NSMakeSize(minimumSize.x, minimumSize.y)];
```

Comment:
Does this consider the window style of no resize? Does MAC not allow windows to disable resizing?

Reply:
That's all automatically handled. Just like the other platforms, macOS will resize the window if the application calls `setSize` but it doesn't not let users resize the window. You can still freely change the min/max size while the window has a non-resizable style.